### ISSUE TITLE
Moroes phase 2 doesn't turn hostile

### ISSUE THREAD

**Issue by MargaretClam** [slowtorta/turtlewow-bug-tracker#5863]
**Please, provide a clear description what the bug is:**
Phase 2 Moroes doesn't engage in combat after talking to him (he yelled the usual message, but nothing after that), stays non-targetable.
![WoW_qafp3m3fnK](https://github.com/slowtorta/turtlewow-bug-tracker/assets/148210026/7913a37c-e136-406f-ada4-28b1b65f3064)

**Steps to reproduce the behavior with as much detail as possible:**

1. Engage with phase 1 Moroes, defeat him.
2. He doesn't continue to the phase 2, have to defeat him once again (this time everything works, he continues to the next room)
3. In the next room he is interact-able, but doesn't engage in combat, nor can he be talked with again.

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=61226

**Expected behavior. Describe how it should work:**
Moroes should engage in combat after talking with him and the usual message, be targetable with spells, killable.

**Comment by twowTurtReynolds** [comment]
The boss works for most people with these replication steps. THere has to be something was done differently with your team. Can you elaborate the replication steps?

**Comment by MargaretClam** [comment]
1st try we got the "normal" phase 1 bug, so we tried again.
2nd try we pulled the boss normally and killed it in phase 1 normally, but slow DPS no dots. Then he became green once at 10% HP and aggro'ed our Moonkin (still as a green NPC). We defeated him mid-speech, he teleported to the phase 2 stage.
We cleared trash up to him, our tank talked to him, Moroes did his speech, but stayed green and is no longer able to be interacted with.
